src/EditorFeatures/Core/Implementation/LanguageServer/VisualStudioLspWorkspaceRegistrationService.cs

```diff
@@ -5,13 +5,12 @@
 using System;
 using System.Collections.Immutable;
 using System.Composition;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Internal.Log;
 using Microsoft.CodeAnalysis.LanguageServer;
 using Logger = Microsoft.CodeAnalysis.Internal.Log.Logger;
 
-namespace Microsoft.VisualStudio.LanguageServices.Implementation.LanguageClient
+namespace Microsoft.CodeAnalysis.Editor.Implementation.LanguageClient
 {
     [Export(typeof(ILspWorkspaceRegistrationService)), Shared]
     internal class VisualStudioLspWorkspaceRegistrationService : ILspWorkspaceRegistrationService
```

Comment:
Does this need to be renamed?

Reply:
(it's not clear to me why it got a "VisualStudio" prefix in the first place...?)